<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::Window</class>
 <widget class="QWidget" name="Otter::Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>27</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="navigationWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,3,1">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="backButton">
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="forwardButton">
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="reloadOrStopButton">
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Otter::AddressWidget" name="addressWidget">
        <property name="placeholderText">
         <string>Enter address or search...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Otter::SearchWidget" name="searchWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::AddressWidget</class>
   <extends>QLineEdit</extends>
   <header>src/ui/AddressWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::SearchWidget</class>
   <extends>QComboBox</extends>
   <header>src/ui/SearchWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
